store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReader.java

```diff
@@ -121,6 +122,7 @@ public static CarbonReaderBuilder builder(String tablePath) {
    */
   public void close() throws IOException {
     validateReader();
+    SDKDetailQueryExecutor.shutdownThreadPool();
```

Comment:
Static one we cannot shutdown, as another CarbonReader might be reading same or different table in same process